donut/modules/feedback/helpers.py

```diff
@@ -54,6 +54,7 @@ def send_to_group(group, data):
     data['group'] = group_id
     data['group_name'] = group
     data['poster'] = "{} Feedback".format(group)
+    data['plain'] = data['msg']
```

Comment:
Seems like `data['msg']` should be set to `''` or `None` since it's currently being used as the HTML content of the message